src/components/Firestore/dialogs/AddCollectionDialog.test.tsx

```diff
@@ -125,9 +125,28 @@ describe('step 2', () => {
     expect(getByLabelText('Field')).not.toBeNull();
   });
 
+  // TODO testing suggests that the button is infact disabed by inspecting the DOM
+  // but triggering a click event still triggers the underlying event. This is no
+  // reproducible in the actual GUI.
+  it.skip('[Save] is disabled if invalid doc-data', async () => {
```

Comment:
Totally can punt this later, but try wrapping dialog in a form (we want this later for nice UX on pressing enter), making buttons type="button" and type="submit",  then doing:

```js
fireEvent.submit(getByText('Save'))
```

This worked for me in clone dialog, and it *may* be smart enough to not submit on a disabled button.

Reply:
Ack, will add this for a quick follow-up tomorrow. Adding to the b/ list